packages/docs/src/web/Grid/Grid.story.tsx

```diff
@@ -101,14 +101,14 @@ export const Fluid = () => {
   const maxBreakpointWidthStyle = maxBreakpointWidth[selectBreakpoint] || {};
 
   return (
-    <>
+    <div className="grid__container">
       <h4>
```

Comment:
it would be nice to use classNames on this `h4` too, something like `<h4 className="grid__title">`